DM_2/config/validationConfig.js

```diff
@@ -0,0 +1 @@
+module.exports = require('express-joi-validation').createValidator({});
```

Comment:
Why do we need a separate export for this?

Reply:
It can be used in different routers, not only in orderRouter, as we have in this hw